Assets/Mirror/Editor/Weaver/Weaver.cs

```diff
@@ -231,15 +187,6 @@ static bool WeaveModule(ModuleDefinition moduleDefinition)
 
                 // We need to do 2 passes, because SyncListStructs might be referenced from other modules, so we must make sure we generate them first.
                 System.Diagnostics.Stopwatch watch = System.Diagnostics.Stopwatch.StartNew();
```

Comment:
```suggestion
```

Reply:
the stopwatch is still being used. 

Removing the comment...